backend/sessions.ts

```diff
@@ -56,16 +58,19 @@ export const getSessionUserHosts = async () => {
   return session;
 }
 
-export const addClientToSession = async (code: string) => {
+// Added clientIdsWithNick 
+export const addClientToSession = async (code: string, nick: string) => {
```

Comment:
and then maybe we just don't pass the nick here but have a function that changes the client data and call that when the user clicks "Let's gooo!" (so we don't have to add the nick as url params)

Reply:
Okay, then we will be adding Clients to the session in Welcome page, before client page. Should we still have addClientToSession in client page ? Because to add the data, userId should be available.

Reply:
yeah let's move that to the welcome page and just have something like an `updateUser()` function

Reply:
Hi Flo, i finished the changes and also added a new component to display the users who typed in their nicks. 
The names are not being updated automatically as QueueItems in the player screen, i am not sure how to manage that.